editoast/src/views/timetable.rs

```diff
@@ -466,8 +481,20 @@ impl Display for TrainId {
             Self::TrainSchedule(id) => write!(f, "{id}"),
             Self::PacedTrainOccurrence {
                 paced_train_id,
-                index,
+                occurrence_ref: PacedTrainOccurrenceRef::BaseOccurrence { index },
             } => write!(f, "{paced_train_id}#{index}"),
+            Self::PacedTrainOccurrence {
+                paced_train_id,
+                occurrence_ref: PacedTrainOccurrenceRef::CreatedException { exception_key },
+            } => write!(f, "{paced_train_id}@{exception_key}@"),
```

Comment:
```suggestion
            } => write!(f, "{paced_train_id}@{exception_key}"),
```

that's peculiar thus source of error